install.sh

```diff
@@ -0,0 +1,877 @@
+#!/bin/sh
+
+set -e
+
+if [ "${DEBUG}" = 1 ]; then 
+    set -x
+fi
+
+# Usage:
+#   curl ... | ENV_VAR=... sh -
```

Comment:
Wondering if we should remove any curl mention, in this script since its not uploaded yet

Reply:
I'll remove it and add it back when that feature is possible.